src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -421,9 +427,11 @@ def register_blob_source_uri_arguments(scope):
     c.reg_arg('destination', options_list=('--destination', '-d'))
 
     with c.arg_group('Download Control') as group:
-        group.reg_arg('validate_content')
         group.reg_arg('max_connections')
 
+        with VersionConstraint(ResourceType.DATA_STORAGE, min_api='2016-05-31') as vc:
```

Comment:
@derekbekoe  Can you please verify this line 432? Does this need to start without any white spaces in the beginning?